### PR TITLE
Update the User Guide

### PR DESCRIPTION
**Pull request by ParthGandhiNUS**
Tabularized the command summary
Reformat-ed the positioning of the different sections
Added TOC for User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # User Guide for **Classify**
 
+## Table of Contents
```
Comment (blackmirag3) [inline docs/UserGuide.md:3]: Nicely done, intuitive idea

**Review by blackmirag3 — APPROVED** [review]
More complete and concise explanations for user functions, well done!

**Comment by alalal47** [comment]
solid formatting, neatly arranged and definitely helps the user more immediately locate info they need, nicely done!

**Comment by alalal47** [comment]
last note, view_subject will be removed from UG next time I update

**Pull request merged**